### PR TITLE
Updated gems with security alerts (rails, nokogiri, jquery)

### PR DESCRIPTION
**Pull request by rewritten**

### CODE REVIEW

**Comment by rewritten** [comment]
Travis complains about something when initializing the Rollbar integration, but I'm not really sure it's safe to skip it.

File: Gemfile [hunk 1]
```diff
@@ -34,20 +34,23 @@ gem 'elasticsearch-model'
...
+# Security
+gem 'nokogiri', '1.6.7.2'
+
 group :development do
   gem "binding_of_caller"
   gem "better_errors"
   gem "rubocop"
   gem "haml-lint"
-  gem 'web-console', '~> 2.0'
+  gem 'web-console', '~> 2.1.3'
```
Comment (enricostano) [inline Gemfile]: any reason this one uses `~>` ? :grimacing:

**Comment by enricostano** [comment]
> Travis complains about something when initializing the Rollbar integration, but I'm not really sure it's safe to skip it.

https://github.com/coopdevs/timeoverflow/blob/master/config/initializers/rollbar.rb

**Comment by rewritten** [comment]
Of course but th exception is raised in
```
~/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/rollbar-1.3.2/lib/rollbar/rake.rb:23:in `patch?'
```

**Comment by enricostano** [comment]
`rollbar (1.3.2)`

try to update it

**Comment by rewritten** [comment]
Found: https://github.com/rollbar/rollbar-gem/pull/397

**Comment by rewritten** [comment]
I wouldn't update without a clear reason (in this case it's in https://github.com/rollbar/rollbar-gem/blob/master/CHANGELOG.md#283)

**Pull request merged**